crates/aptos-crypto/src/ed25519.rs

```diff
@@ -261,15 +272,17 @@ impl PartialEq<Self> for Ed25519PrivateKey {
 
 impl Eq for Ed25519PrivateKey {}
 
-// We could have a distinct kind of validation for the PrivateKey, for
-// ex. checking the derived PublicKey is valid?
+// We could have a distinct kind of validation for the PrivateKey: e.g., checking the derived
+// PublicKey is valid?
 impl TryFrom<&[u8]> for Ed25519PrivateKey {
     type Error = CryptoMaterialError;
 
-    /// Deserialize an Ed25519PrivateKey. This method will also check for key validity.
+    /// Deserialize an Ed25519PrivateKey. This method will NOT check for private key validity beyond
```

Comment:
what changed from check to NOT check?

Reply:
Ugh. Actually, you are right, I was thinking of the Scalar in the "expanded" secret key here, which would've had to be checked for being smaller than the order $\ell$ of the scalar field. Will fix.